src/modules/script.js

```diff
@@ -0,0 +1,80 @@
+const list = JSON.parse(localStorage.getItem('listStorage')) || [];
+const addListItem = document.getElementById('inputs');
+const listContainer = document.querySelector('.todolist');
+export const trashCan = document.querySelectorAll('.delete');
+
+export default class Todolist {
+  constructor(description, complete, index) {
+    this.description = description;
+    this.complete = complete;
+    this.index = index;
+  }
+
+  static createList = () => {
+    const listContainer = document.querySelector('.todolist');
+    const displayList = list.map((list, index) => `<li class="list-item " 
+      id="${index}"><div class="inline">
+      <input id="box-check" type=checkbox class="box-check" name="checkbox">
+      </div> <input type="text" id="xx" class="to-do input-text" 
```

Comment:
It is considered a best practice to have different ids for the elements that need them. Kindly change this line of code so that ids may be unique for every input field. 🙃
![image](https://user-images.githubusercontent.com/80624123/193803020-89967d3a-d343-4e6e-957f-c32b91296eb5.png)